src/msw/rt/notifmsgrt.cpp

```diff
@@ -175,7 +175,7 @@ class wxToastNotifMsgImpl : public wxNotificationMessageImpl
         {
             wxCOMPtr<IToastNotificationFactory> factory;
             hr = rt::GetActivationFactory(RuntimeClass_Windows_UI_Notifications_ToastNotification,
-                IID_IToastNotificationFactory, reinterpret_cast<void**>(&factory));
+                IID_IToastNotificationFactory, reinterpret_cast<void**>(&factory)) ? S_OK : S_FALSE;
```

Comment:
Sorry, but this again seems wrong to me. `SUCCEEDED(S_FALSE)` is true (it's a success HR after all!), so with this change the test below now always passes.

This should be
```suggestion
                IID_IToastNotificationFactory, reinterpret_cast<void**>(&factory)) ? S_OK : E_UNEXPECTED;
```
or something like this (I don't know what does `RoGetActivationFactory()` return in case of failure exactly).

But in fact my preferred solution would be to make `GetActivationFactory()` return `HRESULT`, returning `E_FAIL` or whatever (`CLASS_E_CLASSNOTAVAILABLE` would seem to be fitting) if RT is not available at all. This would make things much less confusing and error-prone IMHO.

Reply:
Oh, indeed. I'll implement your last suggestion.